pkg/k8shandler/configmaps.go

```diff
@@ -46,6 +46,53 @@ type indexSettingsStruct struct {
 	ReplicaShards string
 }
 
+// CreateOrUpdateConfigMap reconciles a configmap
+func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateConfigMap(cm *v1.ConfigMap) error {
+	dpl := elasticsearchRequest.cluster
```

Comment:
this looks like we only use this var if the configmap content has changed, can we move it into that block?